Dockerfile

```diff
@@ -24,12 +28,10 @@ RUN apt-get install -y git jq yarn && \
     apt-get purge -y git jq yarn && \
     apt-get autoremove --yes
 
-RUN adduser --uid 9000 --gecos "" --disabled-password app
-COPY . ./
-RUN chown -R app:app ./
 RUN chown -R app:app $PREFIX
 
 USER app
+COPY . ./
```

Comment:
You're probably right. The motivation was to move rarely changing layers closer to the top so that they were not blown up for no good reason.